swagger.json

```diff
@@ -1118,6 +1118,58 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/log_edits": {
```

Comment:
Should `driver_id` be spinal case according to: https://paper.dropbox.com/doc/API-Best-Practices--AWwpYp50cquK5DmNE9lXnyfWAg-HNfOvJK77p8UIC5BUFO8S

Reply:
nice! we now have some standards!